vllm/worker/model_runner.py

```diff
@@ -1354,6 +1356,9 @@ def execute_model(
             "finished_requests_ids": model_input.finished_requests_ids,
             "request_ids_to_seq_ids": model_input.request_ids_to_seq_ids,
         } if self.has_seqlen_agnostic else {}
+        model_forward_start = torch.cuda.Event(enable_timing=True)
+        model_forward_end = torch.cuda.Event(enable_timing=True)
+        model_forward_start.record()
```

Comment:
Could you please clarify if `torch.cuda.Event()` and `Event.record()` have any overhead when `collect_model_forward_time=False`?

Reply:
Both from the docs it is not supposed (elapsed_time and its sync is the one that can have impact) to have any impact and I ran the benchmark_throughput.py (its measurements seem to have a variance of 5-10%) so did not notice anything signficant. I have run some of our custom internal benchmarking also and confirmed this is not of consequence